### ISSUE TITLE
Segmentation fault (core dumped) in training own data yolo2 darknet

### ISSUE THREAD

**Issue by Aminullah6264** [AlexeyAB/darknet#254]
i have prepare my data according to the instruction given in this link 

https://pjreddie.com/darknet/yolo/

i have downloaded weights also. 
i have 12GB titanX gpu when i run the darknet for training it give this error.
i use following line for training
./darknet detector train cfg/voc.data cfg/yolo-voc.cfg darknet19_448.conv.23

plz how to solve this problem or why it is comming

![image](https://user-images.githubusercontent.com/15268491/32445469-238032ca-c349-11e7-899c-4ff6d128b358.png)

**Comment by AlexeyAB** [comment]
@Aminullah6264 Hi,

1. Did you use this repo or original repo?
2. Did you get this error even for `./darknet detector test ...`?
3. What GPU, CUDNN, OPENCV, ... did you use in your Makefile?
4. Try to compile with GPU=0, CUDNN=0, OPENCV=0, OPENMP=0 in the Makefile, so can train/test works successfully?

**Comment by Aminullah6264** [comment]
Dear @AlexeyAB  
when i run ./darknet detector test
it works for pretrained model but not working for example data training

**Comment by AlexeyAB** [comment]
@Aminullah6264 Try to use this release: https://github.com/AlexeyAB/darknet/releases
It tested on both Windows and Linux for training and detection.

So if it doesn't work for your, then your problem in your environment / libraries (OpenCV, CUDA, cuDNN, ...)

**Comment by xitman47** [comment]
@AlexeyAB hi! 

Unfortunately, I have the same issue on Ubuntu 16. I will try to use earlier release as suggested. Here is the log:

> Loading weights from yolo.weights...
 seen 32
Done!
Learning Rate: 0.001, Momentum: 0.9, Decay: 0.0005
eval: Using default 'voc'
4
"annot load image "data/dog.jpg
Segmentation fault (core dumped)

**Comment by xitman47** [comment]
@AlexeyAB unfortunately, the earlier release does not work either. I googled issue and I thought it is due to not having proper rights on AWS server but changing them did not help. And I am not sure this is the case also cause test does work and save predictions.

I also tried changing images names in the .txt file in many ways but YOLO still cannot load them:

> Loading weights from yolo.weights...
 seen 32
Done!
Learning Rate: 0.001, Momentum: 0.9, Decay: 0.0005
eval: Using default 'voc'
4
"annot load image "data\dog.jpg
Cannot load image "(null)"
Cannot load image "data\eagle.jpg"
sh: 1: Syntax error: word unexpected (expecting ")")
Segmentation fault (core dumped)
